sss/tests/test_geoid.py

```diff
@@ -5,13 +5,52 @@
 import pandas as pd
 import pytest
 
+
 from sss.data import DATA_PATH
 from sss.geoid import GeoID, geo_identifier_creator
 
 
 @pytest.mark.filterwarnings("ignore:Unknown extension is not supported and will be")
-def testgeo_identifier_creator():
+def testgeo_identifier_creator(capsys):
     """Test to see if county_table and cpi_table are joined together."""
+
+    # tests print message is created when we have duplicate rows in CPI table
+    geo_identifier_creator(
+        os.path.join(DATA_PATH, "geoid_data", "SSScounty-place-list_20220720.xlsx"),
+        os.path.join(
+            DATA_PATH,
+            "geoid_data",
+            "StateAbbreviation_Regions_07192022_AKu_duplicate_rows.xlsx",
+        ),
+    )
+
+    out, _ = capsys.readouterr()
+    assert (
+        "Merge sucessful, but new rows were created due to \
+                    duplications in right(CPI) table"
```

Comment:
You actually don't need to use `\` here because python will concatenate lines of strings. So you can replace the `\` with `"` to end the first line string and then start the next line with a `"`. I find that approach more readable.